src/Alley/WP/find-result.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * `find_result()` function
+ *
+ * (c) Alley <info@alley.co>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @package wp-find-one
+ */
+
+namespace Alley\WP;
+
+/**
+ * Returns the first item in the given array if the item is an instance of a
+ * specific class or is a non-object.
+ *
+ * @param string $class   If the result plucked from $results is an object, the
+ *                        name of the class the object must be an instance of
+ *                        for it to be returned.
```

Comment:
Is there any way to provide better typing for this parameter? In the example you used `\WP_Network::class` – is there any way to validate `::class`?

Reply:
Better typing like what? There's nothing invalid about passing the name of a class that doesn't exist, for example.